.gitignore

```diff
@@ -1,2 +1,7 @@
+<<<<<<< HEAD
+node_modules
+client/dist/main.js
+=======
 node_modules
 client/dist/main.js

```

Comment:
It seems like the merge conflict from main was not correctly resolved, it has both versions right now. Avarice could you please double check? @avaboonzaayer

Reply:
Actually, just thought of something. We might also want to add config.js to the gitignore list. We haven't created the file yet but that's where we will store our github token